doc/userguide/support-status.rst

```diff
@@ -0,0 +1,313 @@
+==============
+Support Status
+==============
+
+Levels of Support
+=================
+
+The support tiers detailed below do not represent a binding
+commitment. Instead, they serve as a framework that the OISF employs
+to prioritize features and functionality.
+
+Tier 1
+------
+
+Tier 1 supported items are developed and supported by the Suricata
+core team. These items receive full CI (continuous integration)
+coverage, and functional failures block git merges and releases. Tier
+1 features are enabled by default on platforms that support the
+feature.
+
+Tier 2
+------
+
+Tier 2 supported items are developed and support by the Suricata core
+team, sometimes with help from community. Major functional failures
+block git merges and releases, however less major issues may be
+documented as "known issues" and may go into a release. Tier 1
+features and functionality may be disabled by default.
+
+Community
+---------
+
+When a feature of Suricata is community supported, it means the
+OISF/Suricata development team won’t directly support it. This is to
+avoid overloading the team.
+
+When accepting a feature into the code base anyway, it will come with
+a number of limits and conditions:
+
+* submitter must commit to maintaining it:
+
+  - make sure code compiles and correctly functions after Suricata
+    and/or external (e.g. library) changes.
+  - support users when they encounter problems on the mailinglists and
```

Comment:
```suggestion
  - support users when they encounter problems on the forum and
```

Reply:
Should we add Discord? Sounds too much, considering how volatile messages there can be.